interpreter/src/objects.rs

```diff
@@ -304,7 +305,11 @@ impl PartialOrd for Value {
             (Value::UInt(a), Value::Float(b)) => (*a as f64).partial_cmp(b),
             (Value::Float(a), Value::Int(b)) => a.partial_cmp(&(*b as f64)),
             (Value::Float(a), Value::UInt(b)) => a.partial_cmp(&(*b as f64)),
-            (a, b) => panic!("unable to compare {:?} with {:?}", a, b),
+            (a, b) =>
+            // Compare type ids if types are different.
+            {
+                a.type_of().type_id().partial_cmp(&b.type_of().type_id())
+            }
```

Comment:
This would result in a comparison scheme that doesn't follow the [CEL-spec](https://github.com/google/cel-spec/blob/master/doc/langdef.md#ordering) but if we return `None` here, then the interpreter will return an error indicating that the values cannot be compared.
```suggestion
            (a, b) => None,
```